sass/_CardOverlay.scss

```diff
@@ -75,3 +75,13 @@
   cursor: pointer;
   flex-shrink: 0;
 }
+
+.CoveoResult.coveo-with-card-overlay {
+  transition: min-height .5s;
```

Comment:
Have you tested this in IE? Just to make sure, without the vendor property things could be weird.